openc3/lib/openc3/api/cmd_api.rb

```diff
@@ -538,9 +538,14 @@ def _cmd_implementation(method_name, *args, range_check:, hazardous_check:, raw:
         'log_message' => log_message.to_s,
         'obfuscated_items' => packet['obfuscated_items'].to_s
       }
-      CommandTopic.send_command(command, timeout: timeout, scope: scope)
+      if queue
+        # Pull the command out of the script string, e.g. cmd("INST ABORT")
+        queued = cmd_string.split('("')[1].split('")')[0]
```

Comment:
This means that OBFUSCATE commands are not supported because they are already obfuscated at this point. Supporting obfuscated commands in the queue would be a lot of work.